collect.py

```diff
@@ -111,6 +111,10 @@ def do_something(driver, elem_id=None):
             elem.send_keys('prova@email.it')
         elif input_type == 'password':
             elem.send_keys('aMildlyComplexPasswordIn2017')
+        elif input_type == 'checkbox':
```

Comment:
Since this is the same as a `button` or a `a`, can you move it to `if elem.tag_name in ['button', 'a']:`?

Reply:
@marco-c the ```tag_name``` for ```checkbox``` is ```input``` so I don't think that we can move it there.

Reply:
Right 😄